client/tests/unit/landing.spec.ts

```diff
@@ -64,6 +67,9 @@ describe('landing.vue', () => {
   let wrapper: any
 
   beforeEach(() => {
+    newReqModule.mutateRequestAction('NEW')
```

Comment:
If these mutations only need to be done once in this describe block, they can be put in a "beforeAll()" method.